adagucserverEC/CImageDataWriter.cpp

```diff
@@ -1599,12 +1599,25 @@ int CImageDataWriter::warpImage(CDataSource *dataSource, CDrawImage *drawImage)
               if (contourLine->attr.textcolor.empty() == false) {
                 bilinearSettings.printconcat("textcolor(%s)$", contourLine->attr.textcolor.c_str());
               }
+              if (contourLine->attr.textsize.empty() == false) {
+                bilinearSettings.printconcat("textsize(%s)$", contourLine->attr.textsize.c_str());
+              }
+              if (contourLine->attr.textstrokewidth.empty() == false) {
+                bilinearSettings.printconcat("textstrokewidth(%s)$", contourLine->attr.textstrokewidth.c_str());
+              }
+
+              if (contourLine->attr.textstrokecolor.empty() == false) {
+                bilinearSettings.printconcat("textstrokecolor(%s)$", contourLine->attr.textstrokecolor.c_str());
+              }
```

Comment:
Why is this communicated by putting it into a single string, and later parsing it out again?

Reply:
That is legacy. Now it would be better to directly pass the object. Would say this is technical depth.